source/code/include/playfab/PlayFabPlatformUtils.h

```diff
@@ -0,0 +1,78 @@
+#pragma once
+
+#include <chrono>
+#include <ctime>
+#include <iomanip>
+#include <sstream>
+
+namespace PlayFab
+{
+    typedef std::chrono::time_point<std::chrono::system_clock> TimePoint;
+
+    inline TimePoint GetPlayFabTimePointNow()
```

Comment:
>PlayFabTime [](start = 24, length = 11)

nit:  I think the playfab prefix is unnecessary. recommend just GetTimePointNow()